visualization/app/codeCharta/ui/copyToClipboardButton/copyToClipboardButton.component.spec.ts

```diff
@@ -0,0 +1,31 @@
+import { TestBed } from "@angular/core/testing"
+import { CopyToClipboardButtonComponent } from "./copyToClipboardButton.component"
+import { CopyToClipboardButtonModule } from "./copyToClipboardButton.module"
+
+describe("CopyToClipboardButtonComponent", () => {
+	let component: CopyToClipboardButtonComponent
+
+	beforeEach(() => {
+		const test = TestBed.configureTestingModule({
+			imports: [CopyToClipboardButtonModule]
+		})
+		component = test.createComponent<CopyToClipboardButtonComponent>(CopyToClipboardButtonComponent).componentInstance
+		component["service"].getClipboardText = () => "Magic Monday"
+
+		//clipboard does not exist in jest's JSdom, see https://stackoverflow.com/questions/62351935/how-to-mock-navigator-clipboard-writetext-in-jest
```

Comment:
When we wrote the test, we did not know that clipboard is undefined in jests's JSdom.

This is why we wrote this comment, to document our findings. For you, it may be trivial, for us, it was something new. The next person who reads this test might be a newcomer to MaibornWolff and we thought that she/he might be thankful for this comment.